devtools/chain/config.toml

```diff
@@ -30,6 +30,9 @@ cycles_limit = 99999999
 cycles_price = 1
 interval = 3000
 verifier_list = [ "10f8389d774afdad8755ef8e629e5a154fddc6325a" ]
+private_key = "000000000000000000000000000000001abd6ffdb44427d9e1fcb6f84e7fe7d98f2b5b205b30a94992ec24d94bb0c970"
```

Comment:
What's the difference between it and https://github.com/nervosnetwork/muta/blob/9376d0ef9d92fd55a6483f1f76b1b0d603419f9d/devtools/chain/config.toml#L5?

Reply:
There is no difference, perhaps we should delete the unit test code.